cmd/loop/quote.go

```diff
@@ -44,11 +53,23 @@ func quoteIn(ctx *cli.Context) error {
 	}
 	defer cleanup()
 
-	ctxb := context.Background()
 	quoteReq := &looprpc.QuoteRequest{
 		Amt:        int64(amt),
 		ConfTarget: int32(ctx.Uint64("conf_target")),
 	}
+
+	if ctx.IsSet(lastHopFlag.Name) {
+		lastHopVertex, err := route.NewVertexFromStr(
+			ctx.String(lastHopFlag.Name),
+		)
+		if err != nil {
+			return err
+		}
+
+		quoteReq.LoopInLastHop = lastHopVertex[:]
```

Comment:
Also need to set this for the quote that we get in the [loopin command](https://github.com/lightninglabs/loop/blob/master/cmd/loop/loopin.go#L115), otherwise we won't take last hop into account when we don't call quote directly.